deployment/deploy-core-platform.ps1

```diff
@@ -37,8 +43,32 @@ function CleanHostname([string] $Hostname){
     return $result
 }
 
-# ----- Get AKS Cluster Credentials
-az aks get-credentials --admin --name $AksClusterName --resource-group $AksClusterResourceGroupName --overwrite-existing
+# ----- Get AKS Cluster Credentials into kube context
+if($ArcEnabled){
+    
+    # Arc proxying is now tested on Azure Cloud Shell PW terminal. If running Linux and not in cloudshell: exit
+    if($IsLinux -and !$env:AZUREPS_HOST_ENVIRONMENT)

```

Comment:
Potentially, can be turned into a reusable function as this check is done a few times.

Reply:
Agreed, will optimize as suggested.